core/tee/tadb.c

```diff
@@ -683,6 +683,20 @@ tee_tadb_ta_get_property(struct tee_tadb_ta_read *ta)
 	return &ta->entry.prop;
 }
 
+TEE_Result tee_tadb_get_tag(struct tee_tadb_ta_read *ta, uint8_t *tag,
+			    unsigned int *tag_len)
+{
+	if (*tag_len < sizeof(ta->entry.tag)) {
+		*tag_len = sizeof(ta->entry.tag);
+		return TEE_ERROR_SHORT_BUFFER;
+	}
+
+	memcpy(tag, ta->entry.tag, sizeof(ta->entry.tag));
```

Comment:
From comment of commit _"core: add tee_tadb_get_tag()"_:
> Adds the function tee_tadb_get_tag() which returns a tag that uniquely
identifies a TA.

Are we sure the tags generated in `tadb.c` will be unique per TA?

Reply:
The tag from AES-GCM is as good as a hash.